packages/suite/src/components/wallet/WalletLayout/WalletLayout.tsx

```diff
@@ -41,49 +33,64 @@ export const WalletLayout = ({
     showEmptyHeaderPlaceholder = false,
     title,
     account,
+    isSubpage,
     className,
     children,
 }: WalletLayoutProps) => {
     const { translationString } = useTranslation();
     const l10nTitle = translationString(title);
 
-    useLayout(l10nTitle, AccountTopPanel);
+    useLayout(l10nTitle, PageHeader);
 
     const { status, account: selectedAccount, loader, network } = account;
 
+    let pageContent;
```

Comment:
I don't like this `let`, I would prefer to separate the logic to function and then just call:

`const pageContent = getPageContent(....)`